common/data_refinery_common/models/models.py

```diff
@@ -45,9 +45,21 @@ def __str__ (self):
     source_archive_url = models.CharField(max_length=255)
     source_filename = models.CharField(max_length=255, blank=False)
     source_absolute_file_path = models.CharField(max_length=255)
+    has_raw = models.BooleanField(default=True) # Did this sample have a raw data source?
 
     # Scientific Properties
-    has_raw = models.BooleanField(default=True) # Did this sample have a raw data source?
+    sex = models.CharField(max_length=255, blank=True)
+    age = models.DecimalField(max_length=255, blank=True, max_digits=8, decimal_places=3, null=True)
+    part = models.CharField(max_length=255, blank=True)
+    genotype = models.CharField(max_length=255, blank=True)
+    disease = models.CharField(max_length=255, blank=True)
+    disease_stage = models.CharField(max_length=255, blank=True)
+    cell_line = models.CharField(max_length=255, blank=True)
+    treatment = models.CharField(max_length=255, blank=True)
+    race = models.CharField(max_length=255, blank=True)
+    subject = models.CharField(max_length=255, blank=True)
+    compound = models.CharField(max_length=255, blank=True)
+    time = models.CharField(max_length=255, blank=True)
```

Comment:
The field names are sourced from this comment original ticket: https://github.com/AlexsLemonade/refinebio/issues/165#issuecomment-376684079

With _all_ of these fields, the problem is going to be that they're overloaded in how they're used by submitters, so the more general we can be the better, then at least everything will make some type of sense, even if it's vague. For part, I'd think maybe `organism_part`, but then you could have an `organism_part` that doesn't actually occur organism-wide (ex a tumor), and `subject_part` doesn't always work because cell lines aren't really "subjects" in the sense that we're using it everywhere else. IDK. Time is similarly overloaded to the context of the exposure, but it's often things like exposure-to-a-compound-or-type-of-radiation, but obviously not always.

Reply:
I agree that `part` and `time` are ambiguous. One thing that we will need to do (as @jaclyn-taroni noted somewhere) is explain to people _how_ we harmonized. Is there a way to include some sort of metadata on these fields in a way that would let us make the description available? If we added something like help text with this information, can that be accessed at the API level if we want it to be?

Reply:
@cgreene @jaclyn-taroni do you have a conceptual preference for part:

```python
    part_fields = [
                    # AE
                    'organism part', 
                    'cell type', 
                    'tissue', 
                    'tissue type', 
                    'tissue source', 
                    'tissue origin', 
                    'source tissue', 
                    'tissue subtype', 
                    'tissue/cell type', 
                    'tissue region', 
                    'tissue compartment', 
                    'tissues', 
                    'tissue of origin', 
                    'tissue-type', 
                    'tissue harvested', 
                    'cell/tissue type', 
                    'tissue subregion', 
                    'organ',
                    'characteristic [organism part]',
                    'characteristics [organism part]',

                    # SRA
                    'cell_type'
                    'organismpart',

                    # GEO
                    'isolation source',
                    'tissue sampled',
                    'cell description'

                ]
```
`tissue` maybe?

Time is a lot harder as it's entirely experiment dependent, so the provided and harmonized information is less useful for large-scale analysis, but may be useful while browsing the web interface for samples:

```python
    time_fields = [
                    'time', 
                    'initial time point', 
                    'start time', 
                    'stop time', 
                    'time point', 
                    'sampling time point', 
                    'sampling time',
                    'time post infection'
                ]
```

Reply:
How about `specimen_part` for the first one?

Reply:
I like `time_point` for the second one.